apps/apps_image.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * Twin - A Tiny Window System
+ * Copyright (c) 2024 National Cheng Kung University
+ * All rights reserved.
+ */
+
+#ifndef _APPS_ANIMATION_H_
+#define _APPS_ANIMATION_H_
+
+#include <twin.h>
+
+void apps_image_start(twin_screen_t *screen, const char *name, int x, int y);
```

Comment:
Rename to `apps_animation_start`.

Reply:
This app now supports both static images (JPEG, PNG) and animations (GIF). Would using the term 'animation' be misleading?

Reply:
> This app now supports both static images (JPEG, PNG) and animations (GIF). Would using the term 'animation' be misleading?

The animation can be applied to GIF and animated PNG.